server/models/availability.js

```diff
@@ -0,0 +1,54 @@
+const mongoose = require("mongoose");
+
+
+const dayInformation={
+    isAvaliable:{
+        type:Boolean,
+        default:'true'
+    },
+    startDate:{
+        type:Number,
+        min:0,
+        max:24,
+        required:isAvaliableForDay
+    },
+    endDate:{
+        type:Number,
+        min:0,
+        max:24,
+        required:isAvaliableForDay,
+        validate: [dateValidator, 'Start Date must be less than End Date']
+    }
+}
+function isAvaliableForDay(){
+    if(this.isAvaliable > -1){  
+        return true;
+    }
+    return false;
+}
```

Comment:
I'm a bit confused by this, `isAvailable` is a boolean and hence will either be `0` or `1`, both of which are `> -1`. Am I missing something?

Also, we can scrap this function as per the suggestion above 👍🏼

Reply:
Thanks for the suggestion. Remove this function now the validation is done directly on the start and end time.